{"name":"Log","hostname":"ws-2d68432f-e10f-497c-af96-6c60ac24d43b","pid":2060,"level":50,"req":{"meta":{"requestId":"d57e90f4-71fe-4ac7-b412-3b82ec75b298","userId":null},"url":"/users/me/?include=roles","method":"GET","originalUrl":"/ghost/api/v3/admin/users/me/?include=roles","params":{},"headers":{"host":"localhost:2369","user-agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36","accept":"application/json, text/javascript, */*; q=0.01","accept-encoding":"gzip, deflate, br","accept-language":"en-US,en;q=0.9","app-pragma":"no-cache","content-type":"application/json; charset=UTF-8","cookie":"**REDACTED**","referer":"https://2369-e8f3c2c0-3bd8-44b1-a386-ec254959e14d.ws-eu03.gitpod.io/ghost/","sec-fetch-dest":"empty","sec-fetch-mode":"cors","sec-fetch-site":"same-origin","x-forwarded-for":"10.132.0.14, 10.12.1.83, 10.12.2.80","x-forwarded-host":"2369-e8f3c2c0-3bd8-44b1-a386-ec254959e14d.ws-eu03.gitpod.io:443","x-forwarded-proto":"https","x-ghost-version":"3.40","x-real-ip":"10.132.0.14","x-requested-with":"XMLHttpRequest","x-wsproxy-host":"2369-e8f3c2c0-3bd8-44b1-a386-ec254959e14d.ws-eu03.gitpod.io"},"query":{"include":"roles"}},"res":{"_headers":{"x-powered-by":"Express","cache-control":"no-cache, private, no-store, must-revalidate, max-stale=0, post-check=0, pre-check=0","content-type":"application/json; charset=utf-8","content-length":"321","etag":"W/\"141-392R9Z9ly//md9r285q+tIvApoY\"","vary":"Accept-Encoding"},"statusCode":403,"responseTime":"12ms"},"err":{"id":"0402fe30-4156-11eb-ac45-7dc17c885383","domain":"http://localhost:2369/","code":null,"name":"NoPermissionError","statusCode":403,"level":"normal","message":"Authorization failed","context":"\"Unable to determine the authenticated user or integration. Check that cookies are being passed through if using session authentication.\"","stack":"NoPermissionError: Authorization failed\n    at new NoPermissionError (/workspace/supreme-telegram/ghost/versions/3.40.2/node_modules/ghost-ignition/lib/errors/index.js:134:23)\n    at authorizeAdminApi (/workspace/supreme-telegram/ghost/versions/3.40.2/core/server/services/auth/authorize.js:28:25)\n    at Layer.handle [as handle_request] (/workspace/supreme-telegram/ghost/versions/3.40.2/node_modules/express/lib/router/layer.js:95:5)\n    at next (/workspace/supreme-telegram/ghost/versions/3.40.2/node_modules/express/lib/router/route.js:137:13)\n    at authenticate (/workspace/supreme-telegram/ghost/versions/3.40.2/core/server/services/auth/session/middleware.js:28:13)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)"},"msg":"Authorization failed","time":"2020-12-18T17:25:25.527Z","v":0}
{"name":"Log","hostname":"ws-2d68432f-e10f-497c-af96-6c60ac24d43b","pid":2060,"level":50,"err":{"id":"6aaa2320-4156-11eb-ac45-7dc17c885383","domain":"http://localhost:2369/","code":null,"name":"EmailError","statusCode":500,"level":"normal","message":"Failed to send email. Reason: connect ETIMEDOUT 64.233.167.26:25.","context":"\"Unable to send welcome email, your site will continue to function.\"","help":"\"Please see https://ghost.org/docs/concepts/config/#mail for instructions on configuring email.\"","stack":"EmailError: Failed to send email. Reason: connect ETIMEDOUT 64.233.167.26:25.\n    at EmailError.GhostError (/workspace/supreme-telegram/ghost/versions/3.40.2/node_modules/@tryghost/errors/lib/errors.js:10:26)\n    at new EmailError (/workspace/supreme-telegram/ghost/versions/3.40.2/node_modules/@tryghost/errors/lib/errors.js:34:20)\n    at createMailError (/workspace/supreme-telegram/ghost/versions/3.40.2/core/server/services/mail/GhostMailer.js:52:12)\n    at EventEmitter.<anonymous> (/workspace/supreme-telegram/ghost/versions/3.40.2/core/server/services/mail/GhostMailer.js:119:28)\n    at Object.onceWrapper (events.js:421:26)\n    at EventEmitter.emit (events.js:314:20)\n    at EventEmitter.emit (domain.js:483:12)\n    at DirectMailer.<anonymous> (/workspace/supreme-telegram/ghost/versions/3.40.2/node_modules/directmail/lib/mailer.js:185:42)\n    at SMTPClient.<anonymous> (/workspace/supreme-telegram/ghost/versions/3.40.2/node_modules/directmail/lib/mailer.js:277:9)\n    at Object.onceWrapper (events.js:421:26)\n    at SMTPClient.emit (events.js:314:20)\n    at SMTPClient.EventEmitter.emit (domain.js:483:12)\n    at SMTPClient._onError (/workspace/supreme-telegram/ghost/versions/3.40.2/node_modules/simplesmtp/lib/client.js:373:10)\n    at Socket.emit (events.js:314:20)\n    at Socket.EventEmitter.emit (domain.js:483:12)\n    at emitErrorNT (internal/streams/destroy.js:92:8)"},"msg":"Failed to send email. Reason: connect ETIMEDOUT 64.233.167.26:25.","time":"2020-12-18T17:28:17.749Z","v":0}
